jekyll/_cci2/docker-compose.md

```diff
@@ -15,7 +15,11 @@ This document describes how to install and use `docker-compose`.
 * TOC 
 {:toc}
 
-The `docker-compose` utility is [pre-installed in the CircleCI convenience images][pre-installed] and machine executors. If you're using another image, you can install it into your [primary container][primary-container] during the job execution with the Remote Docker Environment activated by adding the following to your [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file:
+The `docker-compose` utility is [pre-installed in the CircleCI convenience
+images][pre-installed] and machine executors. If you arere using another image, you
```

Comment:
```suggestion
images]and machine executors. If you are using another image, you
```

Reply:
Should there be two instances of [preinstalled] in this sentence?

Reply:
Yeah - the pull request doesn't show it, but there is a markdown link that jekyll compiles in the paragraph below rather than having the link inline :). 

Thanks for catching that typo!